docsite/rst/intro_configuration.rst

```diff
@@ -70,7 +70,7 @@ Actions are pieces of code in ansible that enable things like module execution,
 This is a developer-centric feature that allows low-level extensions around Ansible to be loaded from
 different locations::
 
-   action_plugins = ~/.ansible/plugins/action_plugins/:/usr/share/ansible_plugins/action_plugins
+   action_plugins = ~/.ansible/plugins/action:/usr/share/ansible/plugins/action
```

Comment:
in this particular case/example/context, it has nothing to do with the directories you would have if you were implementing a role which is emphasized by this statement: `low-level extensions around Ansible to be loaded from different locations`.

I understand they are arbitrary but the best example case here is to use what you have as default which documents the default which makes it straightforward step for any developer to know what typically is set and how to modify it to their own taste.